EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -15,15 +14,15 @@ struct MediaPlayerView: View {
             ZStack {
                 viewModel.playerViewHandler
                 
-                if showActionButtons {
+                if viewModel.showActionButtons {
```

Comment:
Actually this var should be something `viewModel.shouldRenderOverlay` or something, cause we are displaying buttons, settings menu etc